### PR TITLE
Fixed formatting and Add API endpoint details

### PR DESCRIPTION
**Pull request by seunbayo**

### CODE REVIEW

File: versioned_docs/version-3.0.0-beta/api/deployment [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+id: deploy
+title: Spheron API
+sidebar_label: deploy
+slug: /api/deployment
+
+---
+
+
+deployments
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/api/deployment:10]: what is this page for?

**Review by rekpero — CHANGES_REQUESTED** [review]
@seunbayo please check my comments

File: versioned_docs/version-3.0.0-beta/api/deployment (hunk 1 above)
Comment (seunbayo) [reply versioned_docs/version-3.0.0-beta/api/deployment:10]: i think I pushed an unfinished branch to these changes,let me finish it up then

**Comment by seunbayo** [comment]
@rekpero  i fixed the formatting and repackaged the API category for more incoming updates

File: versioned_docs/version-3.0.0-beta/api/endpoint.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+👉 On the first visit to the API endpoint,
+
+```
+https://api-v2.spheron.network/status
+```
+
+<strong>
+
+This is what you should see either on postman or on the browser.
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/api/endpoint.md:16]:
```suggestion
This is what you should see either on postman or the browser.
```

File: versioned_docs/version-3.0.0-beta/api/endpoint.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+at the moment, it returns an empty page without any Json response.
```
Comment (rekpero) [inline versioned_docs/version-3.0.0-beta/api/endpoint.md:17]:
```suggestion
At the moment, it returns an empty page without any JSON response.
```

**Review by rekpero — CHANGES_REQUESTED** [review]
@seunbayo small changes needed, please ping once it's done

**Pull request closed without merging**